sdk/textanalytics/ai-text-analytics/test/utils/setup.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { createLiveCredential } from "@azure-tools/test-credential";
+import type { TestProject } from "vitest/node";
+import { EnvVarKeys } from "./constants.js";
+import { CognitiveServicesManagementClient } from "@azure/arm-cognitiveservices";
+import * as MOCKS from "./constants.js";
+
+declare module "vitest" {
+  type MyEnvVarKeys = {
+    [K in (typeof EnvVarKeys)[keyof typeof EnvVarKeys]]: string;
+  };
+  export interface ProvidedContext extends MyEnvVarKeys {}
+}
+
+function assertEnvironmentVariable<T extends Pick<typeof EnvVarKeys, "TEST_MODE">>(
```

Comment:
I'm curious why do we need this overload for `TEST_MODE` specifically? I don't see where this function is called with `TEST_MODE` as a variable

Reply:
The intention here is to force the call site to handle the potential absence of the `TEST_MODE` env var, however, your comment made me realize that it the typing wasn't correct so I corrected it in https://github.com/Azure/azure-sdk-for-js/pull/33531/commits/aa8c21c4ed444b0ecedebb55c3e72af471766247